corehq/apps/api/domain_metadata.py

```diff
@@ -50,6 +51,10 @@ class DomainMetadataResource(HqBaseResource):
     calculated_properties = fields.DictField()
     domain_properties = fields.DictField()
 
+    # using the default resource dispatch function to bypass our authorization for internal use
+    def dispatch(self, request_type, request, **kwargs):
+        return Resource.dispatch(self, request_type, request, **kwargs)
```

Comment:
@orangejenny may have a better answer when she gets back but tastypie will still do user authentication. this only bypasses the domain level authorization I implemented when we restricted APIs to standard and up